### PR TITLE
Define public dataset expiry window in `Tournaments/Official_Support`

### PR DESCRIPTION
**Pull request by hburn7**
Specify that public datasets for automated tools should not be more than 30 days old.

- [x] TC approval
- [x] Outdate translations
- [x] Update last-updated date.

### CODE REVIEW

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:127]: can you give an example of what "dataset" refers to here? data on what? it doesn't mean anything to an outsider

Comment (hburn7) [reply wiki/Tournaments/Official_support/en.md:127]: Dataset essentially means a database of osu! match data which these programs reference in their algorithms. So the context here is "If the program you want to use has a database it builds its performance network from, it must be public".

I think dataset is used twice here to mean two different things. The first use of dataset is talking about what I write above. The second use is, I think, talking about the actual performance network.

There are community tools out there / in development which convert a large amount of osu! match data into some meaningful measure of performance on a per-player basis. i.e. "Player X has 500 rating and Player Z has 200, therefore Player X should win much more often." This rule is in place to allow third parties to validate that the database of osu! data is valid and that the "rating ladder", if you will, isn't being tampered with. Without this rule, hypothetically, the maintainers of such programs could be artificially manipulating the osu! match data fed into the system. They could, in theory, also be artificially boosting or weakening certain players to manipulate who is eligible for certain tournaments.

**Comment by hburn7** [comment]
Added a footnote for dataset, discussed with @ThePooN on data archival period and disclosure of the seed / filter dates. Will look to get TC signoff over the next days.

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:127]: i don't think we need a footnote with a dictionary definition, but just a way to understand what is being said

```suggestion
- If a dataset of osu! multiplayer matches (or otherwise) is used, archives must be publicly available, downloadable, timestamped, and have the most recent archive be no more than 30 days out of date. Archives must remain available for at least 6 months. Documentation must be provided explaining how to reproduce the results using it — including code where applicable.
```

File: wiki/Tournaments/Official_support/en.md
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:303]:
```suggestion
```

File: wiki/Tournaments/Official_support/en.md
Comment (hburn7) [reply wiki/Tournaments/Official_support/en.md]: Agree with this

File: wiki/Tournaments/Official_support/en.md
Comment (Albionthegreat) [inline wiki/Tournaments/Official_support/en.md:127]:
```suggestion
- If a dataset of osu! multiplayer matches, player scores or otherwise is used, archives must be publicly available, downloadable, timestamped, and have the most recent archive be no more than 30 days out of date. Archives must remain available for at least 6 months. Documentation must be provided explaining how to reproduce the results using it — including code where applicable.
```
Though I recognise "or otherwise" is present here as a catch all, I would also like to see player scores mentioend, since  multiplayer matches and user scores combined cover 98% of cases.

Comment (hburn7) [reply wiki/Tournaments/Official_support/en.md:127]: Would a `,` be needed before `or`?

Comment (hburn7) [reply wiki/Tournaments/Official_support/en.md]: Applied

**Review by Albionthegreat — APPROVED** [review]
The changes were discussed and approved by the tournament committee.

**Review by Walavouchey — APPROVED** [review]

**Comment by hburn7** [comment]
Albion said KR translation needs to be outdated still, I won't get to that for a few hours. If auto merge sends it then I'll handle it in another PR.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**
